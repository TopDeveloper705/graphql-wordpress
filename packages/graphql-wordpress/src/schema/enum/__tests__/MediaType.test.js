import MEDIA_TYPE from 'enum/MediaType';

describe('Test enum values', () => {
  test('MEDIA_TYPE values', () => {
    expect(MEDIA_TYPE.getValues()).toMatchSnapshot();
  });
});
