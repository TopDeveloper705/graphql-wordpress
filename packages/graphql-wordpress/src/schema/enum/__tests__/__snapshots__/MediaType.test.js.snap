// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test enum values MEDIA_TYPE values 1`] = `
Array [
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "IMAGE",
    "value": "image",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "VIDEO",
    "value": "video",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "AUDIO",
    "value": "audio",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "APPLICATION",
    "value": "application",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "FILE",
    "value": "file",
  },
]
`;
